lib/src/commonMain/kotlin/sovran.kotlin/Store.kt

```diff
@@ -23,8 +21,7 @@ class Store {
      * it breaks serializability.
      * this queue is specifically for subscriptions
      */
-    private val syncQueue = Executors.newSingleThreadExecutor().asCoroutineDispatcher() +
-            CoroutineName("state.sync.sovran.com")
+    private val syncQueue = CoroutineName("state.sync.sovran.com")
```

Comment:
`CoroutineName` without specifying `ContinuationInterceptor` by default uses `Dispatcher.Default` which uses a shared pool of threads. we purposely force the `syncQueue` and the `updateQueue` to be single thread, so we can achieve serializability within coroutines. that means the queues are consumed in a FIFO manner. changing it to a shared queue would cause tasks running into race condition.